Model/Builders/ConfigRepositoryBuilder.php

```diff
@@ -334,19 +335,21 @@ public function getPaymentNameByMethodCode($methodCode): string
     /**
      * {@inheritDoc}
      */
-    public function getError(array $transaction)
+    public function getError(Order $transaction)
     {
-        if ($transaction['status'] == 'error' && (current($transaction['transactions'])['customer_message']))
+        $status = $transaction->getStatus()->get();
+        $status = 'created';
```

Comment:
What is going on here? The `$status` variable takes its value `$transaction->getStatus()->get(); `and then immediately changes it to `$status = 'created'`

Is it possible to process any other statuses